provider/resource_keycloak_realm.go

```diff
@@ -441,6 +441,148 @@ func resourceKeycloakRealm() *schema.Resource {
 				Type:     schema.TypeMap,
 				Optional: true,
 			},
+
+			// WebAuthn
+			"web_authn_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"acceptable_aaguids": {
+							Type: schema.TypeList,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Optional: true,
+						},
+						"attestation_conveyance_preference": {
```

Comment:
for each attribute which has a set of possible values, can you implement the `ValidateFunc` to ensure this?  the terraform `validation` package has a `StringInSlice` helper function that makes this easier.